src/logs/logger.ts

```diff
@@ -0,0 +1,29 @@
+import winston, { transports } from 'winston';
+
+const { combine, timestamp, json, printf, colorize, align } = winston.format;
+
+export const basicLogger = winston.createLogger({
```

Comment:
Default export logger at the bottom of the file since it's the only function in this file.

```suggestion
const logger = winston.createLogger({
```